components/script/dom/mod.rs

```diff
@@ -324,11 +324,7 @@ pub(crate) mod fontfaceset;
 pub(crate) mod formdata;
 pub(crate) mod formdataevent;
 pub(crate) mod gamepad;
```

Comment:
```suggestion
#[allow(clippy::module_inception)]
pub(crate) mod gamepad;
```
Should we do this instead?

Reply:
Why is this marked as resolved?